app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -285,7 +285,6 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
         positiveButton.visibility = View.GONE
         negativeButton.visibility = View.GONE
         underRevision.visibility = View.VISIBLE
-        footer.visibility = View.VISIBLE
```

Comment:
Yes, you are right.
But this visibility change is already done in `showOtherClassifications()` method.